documentation/assemblies/assembly-external-logging.adoc

```diff
@@ -14,7 +14,7 @@ spec:
   logging:
     type: inline
     loggers:
-      kafka.root.logger.level: "INFO"
+      log4j.logger.kafka: "INFO"
```

Comment:
Looking at https://strimzi.io/docs/operators/master/using.html#external-logging_str
it seems that people might try to configure the TO or UO like this, which would be wrong since they use log4j2. I think we need to move the explanation about the different logging up before we start giving examples on internal vs external logging configuration.